runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/ProcessCollection/ProcessCollection.extraReducers.ts

```diff
@@ -0,0 +1,20 @@
+import { ActionReducerMapBuilder } from '@reduxjs/toolkit';
+
+import { ProcessCollectionState } from './ProcessCollection.state';
+import { createOne } from './ProcessCollection.thunks';
+
+const buildProcessCollectionExtraReducers = (builder: ActionReducerMapBuilder<ProcessCollectionState>) => {
+    builder
+        // CREATE ONE
+        .addCase(createOne.pending, (state) => {
+            state.childrenCollections.isLoading = true;
+        })
+        .addCase(createOne.fulfilled, (state) => {
+            state.childrenCollections.isLoading = false;
+        })
```

Comment:
ideally, here you should update global state and you don't need to fetch all collections once again